js/multiFileUpload.js

```diff
@@ -1,351 +1,390 @@
-(function ($)
+(function ($) {
+
+  "use strict";
+
+  Qubit.multiFileUpload = Qubit.multiFileUpload || {};
+
+  var MultiFileUpload = function (element)
   {
-    Qubit.multiFileUpload = Qubit.multiFileUpload || {};
-
-    Drupal.behaviors.multiFileUpload = {
-      attach: function (context)
-        {
-          // Enable uploader
-          var uploader = new YAHOO.widget.Uploader('uploaderOverlay');
-
-          // Put swf object over "Select files" button
-          var uiLayer = YAHOO.util.Dom.getRegion('selectLink');
-          var overlay = $('#uploaderOverlay');
-          overlay.width(uiLayer.right - uiLayer.left);
-          overlay.height(uiLayer.bottom - uiLayer.top);
-
-          function parseHtmlId(id)
-          {
-            var yuiId = id.match(/([0-9]+)$/).shift();
-
-            if (null != yuiId && !isNaN(parseInt(yuiId)))
-            {
-              return yuiId;
+    this.uppy = new Uppy.Core({ 
+      debug: false,
+      id: 'Uppy-atom',
```

Comment:
Just cosmetic, is the uppercase needed?

Reply:
That was the Uppy way. I have updated it to be all lower case as it should not affect anything.